web_console/pages/job/utils.js

```diff
@@ -0,0 +1,26 @@
+export const FLAppStatus = {
```

Comment:
what about rename this file to `src/utils/job.js`? 'caz it might be used across `pages` and `components`

Reply:
ok